### PR TITLE
Cleaning data updates

### PR DESCRIPTION
**Pull request by RacerBG**
Just some random cleaning data updates based on my crrent load order.

### CODE REVIEW

**Comment by sibir-ine** [comment]
Thanks, can you please list version numbers (& versions where applicable) of each?

**Comment by RacerBG** [comment]
> Thanks, can you please list version numbers (& versions where applicable) of each?

Could you please elaborate? I'm new to this process. :smiley:

**Comment by sibir-ine** [comment]
For each of the plugins, list its version number when you downloaded it. For instance, if you downloaded v2.0 of example mod A & included its cleaning data in the PR, please state that its cleaning corresponds to v2.0 of the mod. Further, if there are multiple versions of the mod available, please specify the version(s) you've provided cleaning data for. For instance, if mod A has option 1 & option 2 available for download, each with the same plugin filename but different contents, & you've only provided cleaning data for option 1, please state this.

**Comment by RacerBG** [comment]
SkyUI - 5.2SE
Unique Uniques SE Re-Ported - 2.3.3
Civil War Aftermath SE - 1.9
Guard Dialogue Overhaul SE (ESL version) - 3.0
Misc Dialogue Edits - 1.9.5.2
SkyTEST - Realistic Animals and Predators SE - 1.65.22
Even Better Quest Objectives SE - EBQO SE - 1.10.0
After the Civil War - Siege Damage Repairs - 2.6.0
The Choice is Yours - 2.7
YOT - Your Own Thoughts SE - First Person Messages - 1.21AE
Beyond Skyrim - Wares of Tamriel SE - 1.5.3
Skyrim Battle Aftermath SE - 1.9
JK's Skyrim - 1.7

File: masterlist.yaml [hunk 1]
```diff
@@ -9458,11 +9458,9 @@ plugins:
...
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3484/'
         name: 'Civil War Aftermath'
-    dirty:
-      - <<: *quickClean
-        crc: 0xD6093F62
-        util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
-        itm: 9
+    clean:
+      - crc: 0x2AEF51C4
+        util: 'SSEEdit v4.1.5f'
```
Comment (sibir-ine) [inline masterlist.yaml]: Double-checked this one, & it looks like you've cleaned this one previously. I get the checksum you got after I clean it:

```suggestion
    dirty:
      - <<: *quickClean
        crc: 0xD6093F62
        util: '[SSEEdit v4.1.5g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
        itm: 9
```

File: masterlist.yaml [hunk 2]
```diff
@@ -14334,12 +14332,8 @@ plugins:
...
         condition: 'version("Guard Dialogue Overhaul.esp", "2.17", >=)'
     clean:
-      - crc: 0x6ABC1129
-        util: 'SSEEdit v4.0.2f'
-      - crc: 0x82FE6344
-        util: 'SSEEdit v4.0.2f'
-      - crc: 0x18AEFA4B
-        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+      - crc: 0x21C5C3DB
+        util: 'SSEEdit v4.1.5f'
```
Comment (sibir-ine) [inline masterlist.yaml]: Two of those checksums are from the other version, so they should be kept.

```suggestion
      # Redux, True GDO Port
      - crc: 0x6ABC1129
        util: 'SSEEdit v4.0.2f'
      # Redux
      - crc: 0x82FE6344
        util: 'SSEEdit v4.0.2f'
      # SE
      - crc: 0x21C5C3DB
        util: 'SSEEdit v4.1.5f'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -15136,14 +15128,12 @@ plugins:
         condition: 'active("YASH2.esp") and not active("BetterQuestObjectives - YASH Patch.esp")'
     tag: [ Text ]
     clean:
-      - crc: 0x1D471A94
-        util: 'SSEEdit v4.0.4c'
-      - crc: 0x1FC5AB07
-        util: 'SSEEdit v4.0.4c'
+      - crc: 0xDF1DDA2D
+        util: 'SSEEdit v4.1.5f'
```
Comment (sibir-ine) [inline masterlist.yaml]: It looks like you provided cleaning data for the USSEP version, the other version should be kept.

```suggestion
      # USSEP
      - crc: 0xDF1DDA2D
        util: 'SSEEdit v4.1.5f'
      # Non-USSEP
      - crc: 0x1FC5AB07
        util: 'SSEEdit v4.0.4c'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -17301,10 +17291,8 @@ plugins:
     tag: [ Text ]
   - name: 'YOT - Your Own Thoughts.esp'
     clean:
-      - crc: 0x63C713B2
-        util: 'SSEEdit v4.0.3h (Hotfix 1)'
-      - crc: 0x5518CDFF
-        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+      - crc: 0x2D1BF884
+        util: 'SSEEdit v4.1.5f'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
      # AE
      - crc: 0x2D1BF884
        util: 'SSEEdit v4.1.5f'
      # SE
      - crc: 0x63C713B2
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
      # SE, Alt. Game Mechanics Vanilla
      - crc: 0x5518CDFF
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
```

File: masterlist.yaml [hunk 5]
```diff
@@ -20883,9 +20871,7 @@ plugins:
     after: [ 'Immersive War and Violence.esp' ]
     clean:
       - crc: 0xC536917E
-        util: 'SSEEdit v4.0.3'
-      - crc: 0x51F9FAE2
-        util: 'SSEEdit v4.0.3'
+        util: 'SSEEdit v4.1.5f'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        util: 'SSEEdit v4.1.5f'
      # No Dragon Attack
      - crc: 0x51F9FAE2
        util: 'SSEEdit v4.0.3'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**